libbpf-tools/sigsnoop.c

```diff
@@ -63,6 +63,39 @@ static const char *sig_name[] = {
 	[29] = "SIGIO",
 	[30] = "SIGPWR",
 	[31] = "SIGSYS",
+	[32] = "N/A",
+	[33] = "N/A",
+	[34] = "SIGRTMIN",
```

Comment:
The man page says that `SIGRTMIN` is not constant.
Maybe:
```c
[SIGRTMIN] = "SIGRTMIN"
[SIGRTMIN+1] = "SIGRTMIN+1"
```

Reply:
Thanks, @chenhengqi , SIGRT{MIN,MAX} is not constant in glibc header, thus, I get these two constant values ​​in Makefile, add two new macro definitions, and replace them in sigsnoop.c. please review, thanks again.